### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN038.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#85] [General]
# https://blog.3place.cn/3place/lmyn/LMYN038.html

https://blog.3place.cn/3place/lmyn/LMYN038.html

**Comment by picklecai** [comment]
一个葫芦6万子，这是隐喻什么呢？

**Comment by picklecai** [comment]
另一个要生儿子
六十个用一千来乘。’ （1.37.8）

刚意识到，这是数学题：1000*60=60000

**Comment by picklecai** [comment]
许多儿子力大无人超过。
他们精力大名扬四海。

神只能保证孩子力大无比，不能保证他们懂事。
